design/legacy-block-storage.md

```diff
@@ -0,0 +1,91 @@
+# Legacy Block Storage with Rook
+
+
+## Overview
+
+### Value of legacy block storage support to Rook
+
+* In order to completely integrate with Kubernetes and utilize Rook's autonomous healing, scaling, and managing features, backend storage systems need to be software-based, distributed (clustered), and capable of being containerized. 
+Not all modern storage systems meet these requirements.
+* Many expensive and proprietary appliance storage silos are still providing services for Enterprise IT infrastructures.
+* Many companies only trust well-certified and time-proven storage solutions for their mission critical applications
```

Comment:
I agree, and I think this Cinder driver proposal (from a Cinder noob's perspective) seems like a pretty efficient way to add support for a lot of legacy backends without a mountain of work in Rook. I think that as long as there are compelling features this design brings when compared with a user configuring a connection to legacy backends themselves manually, I'm in favor.